eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -375,18 +388,21 @@ public void testPollingClient() throws ExecutionException, InterruptedException
             .pollingIntervalMs(pollingIntervalMs)
             .isGracefulMode(false);
 
-    // Initialize and no exception should be thrown.
     EppoClient eppoClient = clientBuilder.buildAndInitAsync().get();
 
+    // Empty config on initialization
     verify(mockHttpClient, times(1)).getAsync(anyString());
     assertFalse(eppoClient.getBooleanAssignment("bool_flag", "subject1", false));
 
-    // Now, return the boolean flag config (bool_flag = true)
-    when(mockHttpClient.get(anyString())).thenReturn(BOOL_FLAG_CONFIG);
-    // Wait the polling interval
-    Thread.sleep(pollingIntervalMs * 2);
```

Comment:
with jitter and the need to complete a number of other async operations, 2x the polling interval isn't guaranteed to be long enough for the operation to complete, and as such, the test has been failing frequently on github actions.